src/Sanitize.php

```diff
@@ -612,6 +633,26 @@ private function get_cache(string $image_url = ''): DataCache
 
         return $this->cache;
     }
+
+    /**
+     * Get a HTTP client
+     */
+    private function get_http_client(): Client
+    {
+        if ($this->http_client === null) {
+            return new FileClient(
```

Comment:
Similarly here.

Would be nice to merge https://github.com/simplepie/simplepie/pull/817 so that the methods can be the same.